customScripts/expansionsetup.gd

```diff
@@ -365,19 +370,19 @@ func setExpansionTraits(person):
 		for i in person.traitstorage:
 			var trait = globals.origins.trait(i)
 			if trait.tags.has('fertility-trait') && trait.tags.has('virilitytrait'):
-				person.add_trait(i)
+				person.add_trait(trait)
```

Comment:
Adding traits is reverted back to a bugged state.

Reply:
checked out from main and changed/added only what I wanted for next commit.

Reply:
abandoned changes for now